editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -500,6 +500,7 @@ TileSetEditor::TileSetEditor(EditorNode *p_editor) {
 	tools[SHAPE_NEW_RECTANGLE]->set_button_group(tg);
 	tools[SHAPE_NEW_RECTANGLE]->set_tooltip(TTR("Create a new rectangle."));
 	tools[SHAPE_NEW_RECTANGLE]->connect("pressed", callable_mp(this, &TileSetEditor::_on_tool_clicked), varray(SHAPE_NEW_RECTANGLE));
+	tools[SHAPE_NEW_RECTANGLE]->set_shortcut(ED_SHORTCUT("tileset_editor/editmode_collision/shape_new_rectangle", TTR("New Rectangle"), KEY_MASK_SHIFT | KEY_R));
```

Comment:
Why are those under `editmode_collision`? It's not a section, it's the name of another shortcut.

Reply:
I think my logic was because it's in the edit mode collision editor that it should have a new section because that's different than the edit mode overall....

Reply:
I just tested and it seems to work, but it's a bit weird to have `tileset_editor/editmode_collision` be both an actual shortcut and a category for three other shortcuts.

Moreover those three buttons are used for the Collision, Occlusion and Navigation editors, so they're not specific to `editmode_collision` IMO.

I'd suggest using:
```
tileset_editor/shape_new_rectangle
tileset_editor/shape_new_polygon
tileset_editor/shape_delete
```

Reply:
Ok, changes submitted =)